include/emqx_ct_helpers.hrl

```diff
@@ -0,0 +1,140 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2020 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%--------------------------------------------------------------------
+
+-define( MQTT_SSL_TWOWAY, [ { cacertfile, filename:join( [ "etc", "certs", "cacert.pem" ] ) },
+							{ verify, verify_peer },
+							{ fail_if_no_peer_cert, true } ] ).
+
+-define( MQTT_SSL_CLIENT_CERTS, [ { keyfile,	filename:join( [ "etc", "certs", "client-key.pem" ]) },
+								  { cacertfile,	filename:join( [ "etc", "certs", "cacert.pem" ]) },
+								  { certfile, 	filename:join( [ "etc", "certs", "client-cert.pem" ] ) } ] ).
+
+-define( TLS_1_3_CIPHERS,
+	[ { versions, [ 'tlsv1.3' ] },
+	  { ciphers,  [ "TLS_AES_256_GCM_SHA384",
```

Comment:
for tlsv1.3 (and only for 1.3), it does not make sense to add other ciphers because the suites are exclusive (no overlapping wither 1.2 or 1.1).

Reply:
or if this is intended for 1.1, 1.2 and 1.3, then we need to list al three versions in line 26

Reply:
Implemented tlsv1.3 exclusive ciphers only for TLS_1_3_CIPHERS